client/src/components/common/Card/cardContent.js

```diff
@@ -24,6 +25,7 @@ CardContent.propTypes = {
   avatar: propTypes.string.isRequired,
```

Comment:
```suggestion
  avatar: PropTypes.string.isRequired,
```
change this here to PropTypes and from above at line 2